lib/builders/Input.Chat.js

```diff
@@ -43,11 +43,24 @@ function addComment (commentsDiv, id, dataPath) {
     .addChildTag('div')
     .addAttribute('class', 'row')
 
-  rowDiv
+  const leftCol = rowDiv
     .addChildTag('div')
     .addAttribute('class', 'col')
+
+  leftCol
+    .addChildTag('div')
+    .addAttribute('class', 'row')
     .content('{{ comment.message }}')
 
+  const timestampRow = leftCol
+    .addChildTag('div')
+    .addAttribute('class', 'row')
+
+  timestampRow
+    .addChildTag('div')
+    .addAttribute('class', 'q-message-stamp')
+    .content('{{ formatDate(comment.created, \'DD/MM/YY HH:mm\') }}')
+
```

Comment:
To clarify why I've formatted this as above:
![comment_before](https://github.com/wmfs/cardscript-to-quasar/assets/75633865/9366a953-ea02-4a78-8be7-70c106f4ea33)
The above picture is with simply adding the date as the 'stamp' attribute (and leaves an unnecessary gap between the date and the text due to the reply button).
With the highlighted (which puts the 'stamp' in the left col with the message) code it now appears as so:
![comment_after](https://github.com/wmfs/cardscript-to-quasar/assets/75633865/1fcaae72-b939-49d0-aa9d-043b09daceae)